app/assets/stylesheets/course/achievement.scss

```diff
@@ -6,8 +6,8 @@
   .achievement-list,
   .achievement-form {
     .image > img {
```

Comment:
on hindsight this should use some more class descriptors to reference the image, maybe .badge or something. thoughts?